src/azure-cli/azure/cli/command_modules/serviceconnector/_params.py

```diff
@@ -107,6 +107,9 @@ def add_new_addon_argument(context, source, target):
     def add_secret_store_argument(context):
         context.argument('key_vault_id', options_list=['--vault-id'], help='The id of key vault to store secret value')
 
+    def add_service_endpoint_argument(context):
+        context.argument('service_endpoint', options_list=['--service-endpoint'], arg_type=get_three_state_flag(), default=None, help='Connect target service by service endpoint')
```

Comment:
If possible, give a brief intro in the help, at whats case users can use the parameter, i.e, the requirements the source and target resource should meet

Reply:
added